redi-connect-front/src/styles/_variables.scss

```diff
@@ -20,6 +20,7 @@ $size-4: 1.25rem; // 20px
 $size-5: 1rem; // 16px
 $size-6: 0.875rem; // 14px
 $size-7: 0.6875rem; // 11px
+$size-8: 0.5rem; // 8px
```

Comment:
Sizes are defined on the default sizes in the design system. Please don't add a new one.If you have a specific size, please hardcode or do something like: 
`$size-5 / 2`

https://deloittedigitalde.invisionapp.com/d/main#/console/19625674/410340233/preview